### DISCUSSION TITLE
Is there a bug with Alt listener ?

### DISCUSSION THREAD

**Discussion by joffreypersia** [alpinejs/alpine#4225] [7. Bugs (archived)]
Hi, 

I am building the shortcut keys of my web app. 

I noticed something. 
```
x-on:keydown.alt.window="console.log('alt')" // works
x-on:keydown.alt.g.window="console.log('alt + g')" // doesn't work
x-on:keydown.meta.window="console.log('cmd ')" // works
x-on:keydown.meta.g.window="console.log('cmd + g')" // works
```

I thought it was a bug on my web app, but I could reproduce the bug here : 
https://codesandbox.io/p/sandbox/alpine-js-forked-8vxq49

Curious to read your guesses :)

**Comment by SimoTod** [comment]
not really. Alpine just uses whatever the native browser events send. alt+g doesn't actually send the g key but `©` (At least in my keyboard). You can see it by adding `x-on:keydown.window="console.log($event)"`, firing the event and checking what the browser adds to the key property

**Comment by ekwoka** [comment]
Unfortunately, it's just how browsers/keyboard/OSs work. some special kinds of combinations do not send a composition of those things in the event, but instead send the thing that that combination makes. Similarly, cmd+click sends an auxclick, not a click, ctrl+click sends a contextmenu, not a click with ctrlkey.

Here, alt+key may not send the key you think you're sending.

**Reply by joffreypersia** [reply]
Thank you guys !

I am building an web app but I can't figure out what shortcut is the best to do something like "something + G" then "P" to move to profile page

Because of all the shortcut keys around, it is difficult to choose one.

**Reply by ekwoka** [reply]
There's some common ones that may align.

But also making it customizable.

You could try to listen for alt+g and alt+ whatever alt+g creates on common layouts. But maybe sticking to cmd+ and cmd+shift+ is better
